connectors/KnowledgeLakeRPA/readme.md

```diff
@@ -0,0 +1,25 @@
+﻿
+## KnowledgeLake RPA
+KnowledgeLake RPA allows you to integrate any Windows, browser, or legacy system with any other application, data source, or custom API without having to make changes to your application or requiring cooperation from the application’s vendor.
+
+
+## Pre-requisites
+* A Microsoft PowerApps or Microsoft Flow plan with custom connector feature
```

Comment:
@SteveCurran Please change it to "A Microsoft Power Apps or Power Automate plan with custom connector feature" - Refer [readme.md](https://github.com/microsoft/PowerPlatformConnectors/blob/master/samples/AzureKeyVault/Readme.md)